### PR TITLE
fix for #6482 - Make sure that the width of the column is a least the size of the label for Semantic Version

### LINKED ISSUE #6482: $PSVersionTable.PSVersion throws error System.ArgumentOutOfRangeException,Microsoft.PowerShell.Commands.OutLineOutputCommand
$PSVersionTable.PSVersion.ToString() and $PSVersionTable still works.

The error can be reproduced on master branch.

Steps to reproduce
------------------

```powershell

$PSVersionTable.PSVersion
out-lineoutput : Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
+ CategoryInfo          : NotSpecified: (:) [out-lineoutput], ArgumentOutOfRangeException
+ FullyQualifiedErrorId : System.ArgumentOutOfRangeException,Microsoft.PowerShell.Commands.OutLineOutputCommand

```

Expected behavior
-----------------

```none
Get the PSVersion string
```

Actual behavior
---------------
Error - System.ArgumentOutOfRangeException,Microsoft.PowerShell.Commands.OutLineOutputCommand

```none

```

Environment data
----------------

```powershell
> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.0.1
PSEdition                      Core
GitCommitId                    v6.0.1-290-g942832c13a3c325851e9873bd25036775c85a18c
OS                             Microsoft Windows 10.0.16299
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

### PR DESCRIPTION
**Pull request by JamesWTruher**
## PR Summary
which in this case is the same as the property name

When the label is too short, an unhelpful error will be emitted
this fixes https://github.com/PowerShell/PowerShell/issues/6482
 
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [x] Issue filed - Issue link: https://github.com/PowerShell/PowerShell/issues/6482
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by stknohg** [comment]
This pull request only partially resolves the problem.

We need to change `TableWriter` class like below.

current :
https://github.com/PowerShell/PowerShell/blob/9bc81ad695c7eba777255a2905aec58adc90daa1/src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs#L292-L310

after change (example):

```csharp
            // add padding for the columns that are shorter
            for (int col = 0; col < scArray.Length; col++)
            {
                int paddingBlanks = _si.columnInfo[validColumnArray[col]].width;
                if (col > 0)
                    paddingBlanks += ScreenInfo.separatorCharacterCount;
                else
                {
                    paddingBlanks += _startColumn;
                }
                int paddingEntries = screenRows - scArray[col].Count;
                if (paddingEntries > 0)
                {
                    for (int j = 0; j < paddingEntries; j++)
                    {
                        scArray[col].Add(col == scArray.Length - 1 ? "" : StringUtil.Padding(paddingBlanks));
                    }
                }
            }
```

File: test/powershell/engine/Basic/SemanticVersion.Tests.ps1 [hunk 1]
```diff
@@ -245,4 +245,10 @@ Describe "SemanticVersion api tests" -Tags 'CI' {
             $des.ToString() | Should -Be $expectedResult
         }
     }
+
+    Context "Formating" {
```
Comment (SteveL-MSFT) [inline test/powershell/engine/Basic/SemanticVersion.Tests.ps1]: Typo: formatting

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: test/powershell/engine/Basic/SemanticVersion.Tests.ps1 (hunk 1 above)
Comment (JamesWTruher) [reply test/powershell/engine/Basic/SemanticVersion.Tests.ps1]: fixed

**Comment by stknohg** [comment]
#6504 will fix this issue.
I retract the previous comment.

**Review by SteveL-MSFT — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**